tools/systemctl

```diff
@@ -85,11 +85,12 @@ SYSTEMCTL_UNITS_COMMON_INTERNAL="bios tntnet@bios fty-outage fty-metric-store
                                 bios-reset-button bios-ssh-last-resort          \
                                 biostimer-compress-logs biostimer-verify-fs     \
                                 biostimer-loghost-rsyslog-netconsole fty-nut    \
-                                biostimer-warranty-metric ifplug-dhcp-autoconf  \
```

Comment:
Removing this name from manageable services is not quite required, and will come back in some form after fty-warranty is completed.

Reply:
once fty-warranty is done it can be replaced. Let's treat it as reminder.

Reply:
Confirmed with @jana-rapava that the fty-warranty is already trying to get into the image (maybe conflicting over the same-named service files - verifying that ATM). It will return under the same name and path, so removing it from `systemctl` now and reviving in next commit is pointless :)